presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -1404,6 +1414,34 @@ void PrestoServer::reportNodeStatus(proxygen::ResponseHandler* downstream) {
   http::sendOkResponse(downstream, json(fetchNodeStatus()));
 }
 
+void PrestoServer::handleGracefulShutdown(const std::vector<std::unique_ptr<folly::IOBuf>>& body , proxygen::ResponseHandler* downstream){
+    std::string requestBody;
+    for (const auto& buf : body) {
+        // Convert IOBuf to fbstring and then to std::string
+        requestBody += buf->moveToFbString().toStdString();
```

Comment:
Can we then use `auto requestBody = body[0]->toString();`?

Reply:
There is no toString() method in folly so I am using moveToFbString() method.